src/meta/src/rpc/metrics.rs

```diff
@@ -740,6 +751,14 @@ impl MetaMetrics {
         )
         .unwrap();
 
+        let opts = histogram_opts!(
```

Comment:
Have we monitored the rate (frequency) of fetching time travel version?

Reply:
Yes, it can be derived from time_travel_version_replay_latency.